### PR TITLE
Added frustum culling support for model component

### LINKED ISSUE #6937: Frustum culling for models
Add for frustum culling for labels (see https://github.com/defold/defold/pull/6445 as a starting point)

### PR DESCRIPTION
**Pull request by JCash**
Fixes #6937 

## PR checklist

* [x] Code
	* [ ] Add engine and/or editor unit tests.
	* [x] New and changed code follows the overall code style of existing code
	* [x] Add comments where needed
* [ ] Documentation
	* [ ] Make sure that API documentation is updated in code comments
	* [ ] Make sure that manuals are updated (in github.com/defold/doc)
* [ ] Prepare pull request and affected issue for automatic release notes generator
	* [ ] Pull request - Write a message that explains what this pull request does. What was the problem? How was it solved? What are the changes to APIs or the new APIs introduced? This message will be used in the generated release notes. Make sure it is well written and understandable for a user of Defold.
	* [ ] Pull request - Write a pull request title that in a sentence summarises what the pull request does. Do not include "Issue-1234 ..." in the title. This text will be used in the generated release notes.
	* [x] Pull request - Link the pull request to the issue(s) it is closing. Use on of the [approved closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
	* [x] Affected issue - Assign the issue to a project. Do not assign the pull request to a project if there is an issue which the pull request closes.
	* [ ] Affected issue - Assign the "breaking change" label to the issue if introducing a breaking change.
	* [ ] Affected issue - Assign the "skip release notes" is the issue should not be included in the generated release notes.

### CODE REVIEW

**Comment by JCash** [comment]
FIxes https://github.com/defold/defold/issues/6937

File: engine/dlib/wscript [hunk 1]
```diff
@@ -71,7 +71,7 @@ def shutdown(sht):
 
     start = time.time()
     while True:
-        if time.time() - start > 5:
+        if time.time() - start > 10:
             error('HTTP server failed to start within 5 seconds')
```
Comment (Jhonnyg) [inline engine/dlib/wscript:75]: This should probably say 10 seconds if the number has changed

File: engine/gamesys/src/gamesys/components/comp_model.cpp [hunk 2]
```diff
@@ -403,8 +406,13 @@ namespace dmGameSystem
             item.m_Model = resource->m_Meshes[i].m_Model;
             item.m_Mesh = resource->m_Meshes[i].m_Mesh;
             item.m_MaterialIndex = resource->m_Meshes[i].m_Mesh->m_MaterialIndex;
+            item.m_AabbMin = item.m_Mesh->m_AabbMin;
+            item.m_AabbMax = item.m_Mesh->m_AabbMax;
             item.m_Enabled = 1;
             component->m_RenderItems.Push(item);
+
+            // printf("AABB: (%f, %f, %f)  (%f, %f, %f)\n", item.m_AabbMin.getX(), item.m_AabbMin.getY(), item.m_AabbMin.getZ(),
+            //         item.m_AabbMax.getX(), item.m_AabbMax.getY(), item.m_AabbMax.getZ());
```
Comment (Jhonnyg) [inline engine/gamesys/src/gamesys/components/comp_model.cpp:415]: Maybe wrap this is in a pragma or add a comment or something if it's useful for debugging

**Review by Jhonnyg — DISMISSED** [review]
Looks good!

**Pull request merged**